build/build-release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+
+readonly REPO_ROOT=$(git rev-parse --show-toplevel)
+
+sudo rkt run --uuid-file-save=rkt.uuid \
```

Comment:
I'm wary of using multi-stage builds still because that requires our Jenkins builder to be running a new enough docker version... which, iirc, some aren't.